temp-tests/retryReads-toggleServerSideJS.yaml

```diff
@@ -0,0 +1,35 @@
+maintenancePlan:
```

Comment:
These files have been added for convenience. The JS toggle tests are much faster than the cluster resizing ones so its more convenient to iterate on the code using results from just `temp-tests` (the directory containing the two test files)